### PR TITLE
setting up namespace for pulp setup experiment

### PR DESCRIPTION
**Pull request by harshad16**
setting up namespace for pulp setup experiment
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Fixes: https://github.com/operate-first/support/issues/252

### CODE REVIEW

File: cluster-scope/base/core/namespaces/thoth-pulp-experiments/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: "Pulp+Thoth-Station: Experimenting with Pulp"
+    openshift.io/requester: thoth-pulp-experiments
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/thoth-pulp-experiments/namespace.yaml:6]: Can we please match the requester annotation to a single OCP user group that owns this namespace?

**Review by tumido — CHANGES_REQUESTED** [review]
/approve

One change requested before we can lgtm and merge this

File: cluster-scope/base/core/namespaces/thoth-pulp-experiments/namespace.yaml (hunk 1 above)
Comment (harshad16) [reply cluster-scope/base/core/namespaces/thoth-pulp-experiments/namespace.yaml]: yes definitely , thank you.

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**